### PR TITLE
[BLOCK-652] Check action trace receipt when setting global sequence

### PR DESCRIPTION
**Pull request by Duncan-Ultra**

### CODE REVIEW

**Review by keisuke-kanao — DISMISSED** [review]

File: cel.go [hunk 1]
```diff
@@ -177,8 +177,13 @@ func NewActionActivation(stepName string, transaction *pbcodec.TransactionTrace,
 		"transaction_id":    transaction.Id,
 		"transaction_index": transaction.Index,
 		"step":              stepName,
-		"global_seq":        trace.Receipt.GlobalSequence,
-		"execution_index":   trace.ExecutionIndex,
+		"global_seq": func() interface{} {
```
Comment (romain-gilles-ultra) [inline cel.go:180]: You should apply this pattern everywhere the Receipt is used in the `cel.go` file.
See for example the `NewActivation` method or `NewTableActivation`

**Comment by romain-gilles-ultra** [comment]
Ok I get it when the action fails there is no `trace.Receipt` good catch it was good in the rest of the code but not in `cel.go`
But maybe we should fix `cel.go` everywhere

**Review by romain-gilles-ultra — CHANGES_REQUESTED** [review]
I think we should fix `cel.go` everywhere

File: cel.go (hunk 1 above)
Comment (Duncan-Ultra) [reply cel.go:199]: Adding check for `trace.Receipt` in more place, some is not really matter like dbops will not present in failed transaction so just add as precaution.

**Review by romain-gilles-ultra — APPROVED** [review]
Great :+1:

**Pull request merged**